pkg/config/piped.go

```diff
@@ -210,10 +224,22 @@ type PipedGit struct {
 	// The path to the private ssh key file.
 	// This will be used to clone the source code of the specified git repositories.
 	SSHKeyFile string `json:"sshKeyFile"`
+	// Base64 encoded string of ssh-key.
+	SSHKeyData string `json:"sshKeyData"`
 }
 
 func (g PipedGit) ShouldConfigureSSHConfig() bool {
-	return g.SSHKeyFile != ""
+	return g.SSHKeyData != "" || g.SSHKeyFile != ""
+}
+
+func (g PipedGit) LoadSSHKey() ([]byte, error) {
```

Comment:
I feel like what both are set is also a kind of problem. Just in case, don't we need to check if `g.SSHKeyData != "" && g.SSHKeyFile != ""`?

Reply:
Sure. Applied this. 👍